huntsman/camera/__init__.py

```diff
@@ -152,10 +152,8 @@ def create_distributed_cameras(camera_info, logger=None):
     for cam_name, cam_uri in camera_uris.items():
         logger.debug('Creating camera: {}'.format(cam_name))
         cam = PyroCamera(port=cam_name, uri=cam_uri)
-        is_primary = ''
         if primary_id == cam.uid or primary_id == cam.name:
             cam.is_primary = True
-            is_primary = ' [Primary]'
```

Comment:
@AnthonyHorton did you intend to alter the camera string property? This was unused.

Reply:
ISTR there being an `is_primary` variable being used in this area of code at some point. I think this line is an orphaned code fragment from times past.

Reply:
This exists in POCS when the camera is created:

https://github.com/panoptes/POCS/blob/5dbd821b582eedac992da9124defcacc03b0e09a/pocs/camera/__init__.py#L183-L190